python/ray/autoscaler/v2/tests/test_utils.py

```diff
@@ -510,7 +510,7 @@ def test_cluster_status_formatter():
  127.0.0.3: worker_node, starting ray
 Recent failures:
  worker_node: LaunchFailed (latest_attempt: 02:46:40) - Insufficient capacity
- worker_node: RayletUnexpectedlyDied (ip: 127.0.0.5)
+ worker_node: RayletTerminated (ip: 127.0.0.5)
```

Comment:
NodeTerminated? does user even know Raylet

Reply:
agree, might be better.